nemo/collections/asr/models/rnnt_models.py

```diff
@@ -996,7 +996,7 @@ def decoder_joint(self):
         return RNNTDecoderJoint(self.decoder, self.joint)
 
     @classmethod
-    def list_available_models(cls) -> Optional[PretrainedModelInfo]:
+    def list_available_models(cls) -> List[PretrainedModelInfo]:
```

Comment:
Same here and above

Reply:
If I correctly understood our conversation in Slack, it's ok (there were no quotes here, actually `List` is returned)